### PR TITLE
gene prioritisation component tabs

### PR DESCRIPTION
**Pull request by chinmehta**
[x] Made a separate section under study locus page for Genes Prioritisation Tabs from page component.
[x] Gene Prioritisation Tabs section component take input for making separate gql call.
[x] Added Skeleton for loading state.
[x] Part of issue #: https://github.com/opentargets/issues/issues/2662

### CODE REVIEW

File: apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    id: 'gene.symbol',
+    label: 'Gene',
+    comparator: (a, b) => ascending(a.gene.symbol, b.gene.symbol),
+    renderCell: d => <Link to={`/gene/${d.gene.id}`}>{d.gene.symbol}</Link>,
+  },
+  {
+    id: 'phenotypeId',
+    label: 'Molecular trait',
+    // renderCell: d => (d.phenotypeId !== d.gene.id ? d.phenotypeId : null),
+  },
```
Comment (carcruz) [inline apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx:31]: Remove commented code

File: apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  {
+    id: 'h3',
+    label: 'H3',
+    tooltip: (
+      <React.Fragment>
+        Posterior probability that the signals <strong>do not</strong>{' '}
+        colocalise
+      </React.Fragment>
+    ),
```
Comment (carcruz) [inline apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx:65]: Fragment component

File: apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+  );
+
+  if(queryResult) {
+    qtlColocDownloadData = getDownloadData(queryResult.qtlColocalisation);
+    qtlColocalisation = queryResult.qtlColocalisation;
+    studyInfo = queryResult.studyInfo;
+  }
+
+  return (
+    <React.Fragment>
```
Comment (carcruz) [inline apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx:115]: Fragment component

File: apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+      <SectionHeading
+        heading={
+          <div id="coloc">
+            Gene prioritisation using colocalisation analysis
+          </div>
+        }
+        subheading={
+          <React.Fragment>
```
Comment (carcruz) [inline apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx:123]: Fragment component

**Review by carcruz — CHANGES_REQUESTED** [review]

File: apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx (hunk 1 above)
Comment (chinmehta) [reply apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx]: Done

File: apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx (hunk 2 above)
Comment (chinmehta) [reply apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx]: Done

File: apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx (hunk 3 above)
Comment (chinmehta) [reply apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx]: Done

File: apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx (hunk 4 above)
Comment (chinmehta) [reply apps/genetics/src/sections/studyLocus/GenePrioritisationTabs/GenePrioritisationTabs.jsx]: Done

**Review by carcruz — APPROVED** [review]

**Pull request merged**